pkg/storegateway/indexcache/tracing.go

```diff
@@ -113,6 +113,29 @@ func (t *TracingIndexCache) FetchSeries(ctx context.Context, userID string, bloc
 	return data, found
 }
 
+func (t *TracingIndexCache) StoreSeriesForPostings(ctx context.Context, userID string, blockID ulid.ULID, matchersKey LabelMatchersKey, shard *sharding.ShardSelector, postingsKey PostingsKey, v []byte) {
+	t.c.StoreSeriesForPostings(ctx, userID, blockID, matchersKey, shard, postingsKey, v)
+}
+
+func (t *TracingIndexCache) FetchSeriesForPostings(ctx context.Context, userID string, blockID ulid.ULID, matchersKey LabelMatchersKey, shard *sharding.ShardSelector, postingsKey PostingsKey) ([]byte, bool) {
+	t0 := time.Now()
+	data, found := t.c.FetchSeriesForPostings(ctx, userID, blockID, matchersKey, shard, postingsKey)
+
+	spanLogger := spanlogger.FromContext(ctx, t.logger)
+	level.Debug(spanLogger).Log(
+		"msg", "IndexCache.FetchSeriesForPostings",
+		"requested key", matchersKey,
+		"shard", shardKey(shard),
+		"found", found,
+		"time elapsed", time.Since(t0),
+		"returned bytes", len(data),
+		"user_id", userID,
+		"postings_key", postingsKey,
```

Comment:
[nit] Just for consistency with the other fields.

```suggestion
		"postings key", postingsKey,
```